localStorage-example/README.md

```diff
@@ -0,0 +1,23 @@
+### Demonstrates how to get & set localStorage items via a Zendesk App
+
```

Comment:
yea sorry i didn't realize there were 2 apps. so yea same feedback from both README, basically just consistent with the rest of the repo. Otherwise all good